### PR TITLE
feat: RPC - `starknet_chainId`

### PR DESCRIPTION
**Pull request by dpinones**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #203 

## What is the new behavior?

- add `starknet_chainId` rpc endpoint.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No 

## Other information

Response example:
Screenshot from 2023-05-14 14-08-49

### CODE REVIEW

**Review by drspacemn — APPROVED** [review]
hex encoded return

File: tests/tests/test-rpc/constants.ts [hunk 1]
```diff
@@ -19,3 +19,6 @@ export const FEE_TOKEN_ADDRESS =
   "0x040e59c2c182a58fb0a74349bfa4769cbbcba32547591dd3fb1def8623997d00";
 export const TOKEN_CLASS_HASH =
   "0x0000000000000000000000000000000000000000000000000000000000010000";
+
+// Starknet testnet SN_GOERLI
+export const CHAIN_ID_STARKNET_TESTNET = "1536727068981429685321";
```
Comment (drspacemn) [inline tests/tests/test-rpc/constants.ts:24]: Looks like current tooling returns the hex encoding:
"0x534e5f474f45524c49"

File: crates/client/rpc/src/lib.rs [hunk 2]
```diff
@@ -477,6 +477,16 @@ where
...
     }
+
+    /// Returns the chain id.
+    fn get_chain_id(&self) -> RpcResult<String> {
+        let hash = self.client.info().best_hash;
+        let res = self.client.runtime_api().chain_id(hash).map_err(|_| {
+            error!("fetch runtime chain id failed");
+            StarknetRpcApiError::InternalServerError
+        })?;
+        Ok(res.to_string())
```
Comment (drspacemn) [inline crates/client/rpc/src/lib.rs:488]: Can we hex encode this return?

Ret - "0x534e5f474f45524c49"

**Review by abdelhamidbakhta — APPROVED** [review]
lgtm

**Pull request merged**